rasa/core/processor.py

```diff
@@ -105,9 +105,12 @@ def _load_model(
         model_path: Union[Text, Path]
     ) -> Tuple[Text, ModelMetadata, GraphRunner]:
         """Unpacks a model from a given path using the graph model loader."""
-        model_tar = get_latest_model(model_path)
-        if not model_tar:
-            raise ModelNotFound(f"No model found at path '{model_path}'.")
+        if os.path.isfile(model_path):
+            model_tar = model_path
+        else:
+            model_tar = get_latest_model(model_path)
+            if not model_tar:
+                raise ModelNotFound(f"No model found at path '{model_path}'.")
```

Comment:
should this be a separate method? In my opinion we should avoid that we have a similar bug in other modules using `get_latest_model`

Reply:
I was going to change `get_latest_model` but looking at it, its pretty clear in its function - I think I just miss-used it here.

Reply:
Could we just raise in `get_latest_model` if what is passed is already a file? I don't find that behaviour of falling back to the dirname in that case is that obvious

Reply:
> I don't find that behaviour of falling back to the dirname in that case is that obvious

I also think the behaviour is weird. However I didn't want to affect other uses of this method in the codebase this close to the release. So I opted to just fixing how we load in the `Processor`.